go/vt/vtadmin/vtctldclient/proxy.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"vitess.io/vitess/go/trace"
 	"vitess.io/vitess/go/vt/grpcclient"
+	"vitess.io/vitess/go/vt/orchestrator/external/golib/log"
```

Comment:
wrong log lib :) (vscode gets me with this one _constantly_); you want `go/vt/log`

Reply:
ARGH thanks